bcipy/tasks/session_data.py

```diff
@@ -0,0 +1,184 @@
+"""Module for functionality related to session-related data."""
+from typing import Dict, List
+from collections import Counter
+
+
+class StimSequence:
+    """Represents a sequence of stimuli."""
+
+    def __init__(self,
+                 stimuli: List[str],
+                 eeg_len: int,
```

Comment:
We could make this a little more generic. `data_len`.

Reply:
I was looking through the usage and what this is referring to is the number of channels in the EEG data. This is going to be the same value for all Inquiries, so I'm not sure it makes sense to store this here. If anything it should be a Session variable, but I'm wondering if it even pertains to sessions at all. It seems like the concern of session-related data should be around what was presented and how decisions were made. Maybe this parameter should be removed. Thoughts?

Reply:
I agree, let's remove it. I think it was used for debugging at one point.